Doppler.ContactPolicies.Business.Logic/Services/ContactPoliciesService.cs

```diff
@@ -0,0 +1,23 @@
+using System.Threading.Tasks;
+using Doppler.ContactPolicies.Data.Access.Entities;
+using Doppler.ContactPolicies.Data.Access.Repositories.ContactPoliciesSettings;
+
+namespace Doppler.ContactPolicies.Business.Logic.Services
+{
+    public class ContactPoliciesService : IContactPoliciesService
+    {
+        private readonly IContactPoliciesSettingsRepository _settingsRepository;
+
+        public ContactPoliciesService(IContactPoliciesSettingsRepository settingsRepository)
+        {
+            _settingsRepository = settingsRepository;
+        }
+
+        public async Task<ContactPoliciesSettings> GetContactPoliciesSettingsAsync(string accountName)
+        {
+            var contactPoliciesSettings =
+                await _settingsRepository.GetContactPoliciesSettingsAsync(accountName);
+            return contactPoliciesSettings;
+        }
+    }
```

Comment:
This layer is redundant, I would prefer not to have this, but, if you really like it, I can accept it.